code/modules/cargo/packs.dm

```diff
@@ -909,7 +909,7 @@
 
 /datum/supply_pack/science/bz_canister
 	name = "BZ Canister"
-	cost = 4000
+	cost = 2000
```

Comment:
People will always pick plasma over BZ since bz would take effort to buy it at cargo